input.py

```diff
@@ -0,0 +1,32 @@
+from typing import Counter
+from unittest.result import failfast
+from spellchecker import SpellChecker
+
+
+def function(sting):
+    userList = sting.split()
+    Counter=0
+    spell=SpellChecker()
+    misspelled=spell.unknown(userList)
+    for word in misspelled:
+        Counter = Counter +1
+    if(Counter!=0):
+        print ("Your input have error in it.... Exiting")
+        return None
+    else: 
+        

```

Comment:
Remove needless space. Should we think about installing a python formatter? I made one here https://github.com/holtzmak/Git-Pre-Commit-and-IntelliJ-Settings

Reply:
If space is a problem then we can definitely get rid of it